schema/cassandra/temporal/versioned/v1.3/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "1.3",
```

Comment:
1.2 is not released yet

Reply:
Oh yeah, I forgot, we agreed to update existing version (instead of creating new one) until release.